vmdb/app/controllers/application_controller.rb

```diff
@@ -52,7 +52,11 @@ class ApplicationController < ActionController::Base
   #   CimBaseStorageExtentController => CimBaseStorageExtent
   #   OntapFileShareController        => OntapFileShare
   def self.model
-    @model ||= self.name[0..-11].constantize
+    @model ||= model_name.constantize
+  end
+
+  def self.model_name
+    name[0..-11]
```

Comment:
Yes, this will always be Controller. 
I'm OK with either approach to get the model name